src/components/auth/profile/DonationTab.tsx

```diff
@@ -75,16 +79,30 @@ const Root = styled('div')(({ theme }) => ({
   },
 }))
 
+//Sum the active donations
+function recurreningDonationsSum(donations: RecurringDonationResponse[] | undefined) {
```

Comment:
Typo here! **recurrening** instead of **recurring**

Reply:
fixed